frontend/app/tsconfig.json

```diff
@@ -1,27 +1,11 @@
 {
-  "include": ["src/*", "../lib/src/declarations.d.ts"],
+  "root": true,
+  "extends": "@streamlit/typescript-config/react.tsconfig.json",
   "compilerOptions": {
-    "allowJs": true,
```

Comment:
Are we removing all these compilerOptions because they are on by default or...?

Reply:
Yea they pull from tsconfig and compilerOptions are merged. So you can look at react.config (which has the jsx param), and also the base.config (which contains everything else.